test/gmp_tcong.rb

```diff
@@ -1,6 +1,6 @@
 require File.expand_path(File.join(File.dirname(__FILE__), 'test_helper'))
 
-class GMP_TCong < Test::Unit::TestCase
+class GMP_TCong < MiniTest::Test
```

Comment:
Cool - this seemed to be the easiest way to ensure the tests run in all versions of ruby from 1.9 to 2.4 :)